app/src/main/java/com/google/android/ground/ui/offlineareas/OfflineAreasViewModel.kt

```diff
@@ -49,7 +49,8 @@ internal constructor(private val offlineAreaRepository: OfflineAreaRepository) :
   val showNoAreasMessage: LiveData<Boolean>
   val showProgressSpinner: LiveData<Boolean>
 
-  private val _navigateToOfflineAreaSelector = MutableSharedFlow<Unit>(replay = 1)
+  private val _navigateToOfflineAreaSelector =
+    MutableSharedFlow<Unit>(extraBufferCapacity = 1, replay = 0)
```

Comment:
Why did we make this change?

Reply:
Fixes https://github.com/google/ground-android/issues/3019